internal/pluck.go

```diff
@@ -1,4 +1,4 @@
-package pluck
+package internal
```

Comment:
I moved everything from pluck to internal, since we don't need the `internal/pluck` package to import stuff into `main.go`